### PR TITLE
Ensure compatibility with PHP 7.2 and 5.6

### PR DESCRIPTION
**Pull request by RedEnchilada**
#15 

This also includes some fixes to the project setup to make it kind of possible to run CI against it.

### CODE REVIEW

**Comment by healthlabs-bugman** [comment]
Paging @juanmrad to review the front-end work in this pull request!
Paging @ramsey-medology to review the back-end work in this pull request!
Paging @awesomeousPrime to review the testing work in this pull request!

I couldn't recognize who these files were relevant to:
.circleci/config.yml
.gitignore
behat.yml
bin/composer
bin/init_project
bin/install_composer
bin/linux-sed
bin/php
docker/docker-compose.yml

**Comment by RedEnchilada** [comment]
I think this is ready to review now

File: web/index.php [hunk 1]
```diff
@@ -97,6 +97,87 @@
...
+                    '<span>Houston, TX, USA</span>' +
+                '</div>' +
+            '</div>';
+
+        google = {maps: {
+            places: {Autocomplete: AutocompleteMock},
+            event: {addListener: AddMockListener}
+        }};
+
+        function AutocompleteMock(element) {
```
Comment (awesomeousPrime) [inline web/index.php]: Please add docblocks to these functions.

**Comment by ramsey-medology** [comment]
Shouldn't there be a CircleCI check that's passing?

**Comment by RedEnchilada** [comment]
There should......

**Pull request merged**